tests/test_switch.py

```
@@ -1100,13 +1106,165 @@ def test_extra_state_attributes(self, mock_coordinator: MagicMock) -> None:
+        ``PolicyBasedRoute`` here is the point: hand-built dicts that omit the
+        sibling alias key do not reproduce real coordinator data.
+        """
+        route = PolicyBasedRoute.model_validate(
+            {
+                "_id": "route4",
+                "description": "Serialized Route",
+                "enabled": True,
+                **route_kwargs,
+            }
+        )
+        route_data = UnifiBaseCoordinator._model_to_dict(mock_coordinator, route)
```

Pylint (4.0.7)

[warning] Access to a protected member _model_to_dict of a client class

(W0212)